test/bigml/test/api/utils.clj

```diff
@@ -0,0 +1,56 @@
+;; Copyright 2016 BigML
+;; Licensed under the Apache License, Version 2.0
+;; http://www.apache.org/licenses/LICENSE-2.0
+
+(ns bigml.test.api.utils
+  "Contains a few facilities especially meant tor testing"
+  (:require (bigml.api [core :as api])))
+
+(defn authenticated-connection
+  "Returns a properly authenticated connection.
+  Credentials shall be provided through the BIGML_USERNAME
+  and BIGML_API_KEY environment variables"
+  []
+  (let [username (System/getenv "BIGML_USERNAME")
+        api-key (System/getenv "BIGML_API_KEY")]
+    (api/make-connection username api-key)))
+
+(defn- do-verb
```

Comment:
Thanks for both comments. I think I can take the suggestion of using a straight map, which is the simplest thing to do. On the other hand, taking the simplest path is not really useful to understand. Here I can only notice two things:

- if I were to define a macro --  since macros are the way to reduce boilerplate as Jao suggests -- this would still not spare me the use of `ns-resolve` or other equivalent means such as `ns-publics` (of course I could always do `(list (symbol (str "bigml.api." (name type) "/" (name verb))))` but *this* would be seriously hacky)

- it is not clear to me why `ns-resolve` is a hack, since it seems an orderly and perfectly legal way to dynamically refer a symbol across namespace boundaries (actually, it is the only one; and it is not like `ns-resolve` is meant for something else and I am stretching it to unnatural ends). When Hickey introduced the bunch of functions to manipulate namespaces, he liked to talk about having `first-class namespaces` -- it does not sound like a hack to me.

That's just a way of reasoning... if you can provide your thoughts, that will be highly appreciated.